src/test/resources/pages/InstructorCourseEnrollPage.html

```diff
@@ -1,34 +1,293 @@
-<div class="container" id="mainContent">
```

Comment:
They are actually added because I used god mode to regenerate them, god mode can be quite erratic sometimes and add in odd indentations or slashes etc, should I manually revert those?